### PR TITLE
fix: plot update; default handler arguments to match old behavior

### PR DESCRIPTION
**Pull request by jokasimr**
Tried to run the example notebook and no figure was produced because the default config of the merge handler was to merge five messages, but the example only produces fewer messages so the workflow was never run.

Also encountered some error with the figure update function. Don't know what that was about, but now it seems fixed.

### CODE REVIEW

File: src/beamlime/applications/handlers.py [hunk 1]
```diff
@@ -66,8 +66,9 @@ class DataAssembler(HandlerInterface):
 
     def __init__(
         self,
-        merge_every_nth: MergeMessageCountInterval = float('inf'),
-        max_seconds_between_messages: MergeMessageTimeInterval = 5,
+        *,
+        merge_every_nth: MergeMessageCountInterval = 1,
+        max_seconds_between_messages: MergeMessageTimeInterval = float('inf'),
```
Comment (jokasimr) [inline src/beamlime/applications/handlers.py:71]: By default, run the workflow every time you receive a message.

File: src/beamlime/applications/handlers.py [hunk 2]
```diff
@@ -135,7 +136,7 @@ def plot_item(self, name: str, data: sc.DataArray) -> None:
             self.artists[name] = next(iter(plot.artists))
             self.figures[name] = plot
         else:
-            figure.update(data, key=self.artists[name])
```
Comment (jokasimr) [inline src/beamlime/applications/handlers.py:138]: is this old plopp code?

Comment (SimonHeybrock) [reply src/beamlime/applications/handlers.py:138]: @nvaytet?

Comment (YooSunYoung) [reply src/beamlime/applications/handlers.py:138]: Probably...?

Comment (YooSunYoung) [reply src/beamlime/applications/handlers.py:138]: Yeah, it was...! I'll update the plopp version in the dependencies : D

**Review by YooSunYoung — APPROVED** [review]

**Pull request merged**